docs/UserGuide.md

```diff
@@ -17,6 +17,7 @@ They can easily look up the list of shows if they need to.
   * [Finding a show: `find`](#finding-a-show-find)
   * [Deleting a show: `delete`](#deleting-a-show-delete)
   * [Editing a show: `edit`](#editing-a-show-edit)
+  * [Editing a comment: `comment`](#editing-a-comment-comment)
```

Comment:
Maybe you could change it to `Commenting a show:` instead?

Reply:
Good point